isomorphe/isomorphe.py

```diff
@@ -68,7 +68,7 @@ def transform(self) -> etree.XSLT:
         return transform
 
 
-class Migrator:
+class Isomorphe:
```

Comment:
I think `Migrator` is still a valid technical name, and avoids repeating isomorphe too much `from isomorphe.isomorphe import Isomorphe`.

Reply:
Was 50-50 on that one. Will revert then.